### ISSUE TITLE
Got sent underground at the middle of the map and stuck there for the rest of the game

### ISSUE THREAD

**Issue by DarkSuo** [ValveSoftware/Dota2-Gameplay#8253]
### Description

Randomly been sent to point 0 of the map at the center under the ground. I was on a teamfight  at bottom jungle and some disable connected on me(i got Hexed by Timbersaw's Vyse) and i got instantly teleported there and couldnt use anything for the rest of the game. Had to abandon so my teammates could use my gold and win.

Match ID 7118303392
Ingame time: 42:28~

### Example Match ID (and possibly Timestamp)

7118303392 - 42:28~

### Screenshots

![image](https://user-images.githubusercontent.com/131493020/233734650-7f26fa3b-409d-4d5b-9263-0d3d374f7603.png)
![image](https://user-images.githubusercontent.com/131493020/233734886-f7832b81-455f-4be6-a5a4-b098ebb9919b.png)

**Comment by commentator24** [comment]
#8147 similar maybe

**Comment by DarkSuo** [comment]
> #8147 similar maybe

Yes, it seems to have to do with being force staffed while disabled. On reddit's bug Mega thread i saw a couple other instances of this with roots and , in my case, hex.
